test/csuite/timestamp_abort/main.c

```diff
@@ -797,12 +898,14 @@ main(int argc, char *argv[])
     /* Copy the data to a separate folder for debugging purpose. */
     testutil_copy_data(home);
 
-    printf("Open database, run recovery and verify content\n");
+    printf("Open database and run recovery\n");
 
     /*
      * Open the connection which forces recovery to be run.
      */
-    testutil_check(wiredtiger_open(NULL, NULL, ENV_CONFIG_REC, &conn));
+    testutil_check(wiredtiger_open(NULL, &my_event, ENV_CONFIG_REC, &conn));
+    printf("Connection open and recovery complete. Verify content\n");
+    usleep(USEC_STAT + 10);
```

Comment:
Maybe a comment why a sleep is needed.  (and why add 10?)

Reply:
Fixed.